k.makoyid/oop_code/Activation_Functions.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+class ActivationFunctions:
+    @staticmethod

```

Comment:
Please add standard relu function. It is commonly used even in modern ML projects

Reply:
according to the theory of the relu activation function, when the parameter alpha(that defines the slope of the function for negative inputs) = 0(now alpha = 0.01), this function will work as a standard relu